content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -0,0 +1,349 @@
+---
+title: kubeadm join
+content_type: concept
+weight: 30
+---
+<!-- overview -->
+Este comando inicializa um nó de processamento do Kubernetes e o associa ao
+cluster.
+
+<!-- body -->
+{{< include "generated/kubeadm_join.md" >}}
+
+### Fluxo do comando `join` {#join-workflow}
+
+O comando `kubeadm join` inicializa um nó de processamento ou um nó da camada
+de gerenciamento e o adiciona ao cluster. Esta ação consiste nos seguintes passos
+para nós de processamento:
+
+1. O kubeadm baixa as informações do cluster necessárias através servidor da API.
```

Comment:
Sounds better to me "O kubeadm baixa as informações necessárias do cluster ...". think about it.

Reply:
Updated!